doc/source/whatsnew/v0.22.0.txt

```diff
@@ -24,6 +24,7 @@ Other Enhancements
 
 - Better support for :func:`Dataframe.style.to_excel` output with the ``xlsxwriter`` engine. (:issue:`16149`)
 - :func:`pandas.tseries.frequencies.to_offset` now accepts leading '+' signs e.g. '+1h'. (:issue:`18171`)
+- :func:`MultiIndex.unique` now supports the ``level=`` argument (:issue:`17896`)
```

Comment:
Can you add briefly what it does, something like ".. to get the unique values of a single index level" ?